stripper/ze_forhidden_void_b5.cfg

```diff
@@ -0,0 +1,660 @@
+;adjust teleport destination
+modify:
+{
+	match:
+	{
+		"targetname" "stage2_zm_end_des"
+	}
+	replace:
+	{
+		"origin" "-798 1112 1893"
+	}
+}
+
+;delay trigger and disable shortcut
```

Comment:
Is there missing comments? Seems to be a ton of unrelated stuff under this comment description.

Reply:
These changes came from the mapper so I have no clue what they do completely.

Reply:
Just comment stuff like that with "uncommented changes from mapper" if you're unable to figure out the purpose of some changes. It's still better than putting stuff under an incorrect comment.